astropy/stats/sigma_clipping.py

```diff
@@ -1,98 +1,189 @@
         The number of standard deviations to use for both the lower and
-        upper clipping limit. These limits are overridden by
-        ``sigma_lower`` and ``sigma_upper``, if input. Defaults to 3.
+        upper clipping limit.  These limits are overridden by
+        ``sigma_lower`` and ``sigma_upper``, if input.  The default is
+        3.
+
     sigma_lower : float or `None`, optional
         The number of standard deviations to use as the lower bound for
-        the clipping limit. If `None` then the value of ``sigma`` is
-        used. Defaults to `None`.
+        the clipping limit.  If `None` then the value of ``sigma`` is
+        used.  The default is `None`.
+
     sigma_upper : float or `None`, optional
         The number of standard deviations to use as the upper bound for
-        the clipping limit. If `None` then the value of ``sigma`` is
-        used. Defaults to `None`.
-    iters : int or `None`, optional
-        The number of iterations to perform sigma clipping, or `None` to
-        clip until convergence is achieved (i.e., continue until the
-        last iteration clips nothing). Defaults to 5.
-    cenfunc : callable, optional
-        The function used to compute the center for the clipping. Must
-        be a callable that takes in a masked array and outputs the
-        central value. Defaults to the median (`numpy.ma.median`).
-    stdfunc : callable, optional
-        The function used to compute the standard deviation about the
-        center. Must be a callable that takes in a masked array and
-        outputs a width estimator. Masked (rejected) pixels are those
-        where::
-
-             deviation < (-sigma_lower * stdfunc(deviation))
-             deviation > (sigma_upper * stdfunc(deviation))
-
-        where::
-
-            deviation = data - cenfunc(data [,axis=int])
-
-        Defaults to the standard deviation (`numpy.std`).
+        the clipping limit.  If `None` then the value of ``sigma`` is
+        used.  The default is `None`.
+
+    maxiters : int or `None`, optional
+        The maximum number of sigma-clipping iterations to perform or
+        `None` to clip until convergence is achieved (i.e., iterate
+        until the last iteration clips nothing).  If convergence is
+        achieved prior to ``maxiters`` iterations, the clipping
+        iterations will stop.  The default is 5.
+
+    cenfunc : {'median', 'mean'} or callable, optional
+        The statistic or callable function/object used to compute the
+        center value for the clipping.  If set to ``'median'`` or
+        ``'mean'`` then having the optional `bottleneck`_ package
+        installed will result in the best performance.  If using a
+        callable function/object and the ``axis`` keyword is used, then
+        it must be callable that can ignore NaNs (e.g. `numpy.nanmean`)
+        and has an ``axis`` keyword to return an array with axis
+        dimension(s) removed.  The default is ``'median'``.
```

Comment:
I know this hasn't changed from before, but it seems wrong to default to median - I always thought of sigma clipping as using the mean (and this is what scipy does). I guess we could open a separate issue to discuss this?